ProjFS.Mac/PrjFSKext/VnodeCache.cpp

```diff
@@ -428,3 +506,30 @@ KEXT_STATIC bool TryInsertOrUpdateEntry_ExclusiveLocked(
     
     return false;
 }
+
+KEXT_STATIC_INLINE void InitCacheStats()
+{
+    atomic_exchange_explicit(&s_cacheStats.cacheEntries, 0U, memory_order_relaxed);
+    
+    for (int32_t i = 0; i < VnodeCacheHealthStat_Count; ++i)
+    {
+        atomic_exchange_explicit(&s_cacheStats.healthStats[i], 0ULL, memory_order_relaxed);
```

Comment:
Adding @jrbriggs in case he has any thoughts on the atomic usage.  John it would be great if you can do a scan of the atomic operations in the file and let us know if you have any thoughts.

For instance this isn't a multi-threaded scenario, but there's concern that if we don't init it this way, it can cause problems later on.

@wilbaker  feel free to add any more thoughts.

Reply:
> but there's concern that if we don't init it this way, it can cause problems later on.

From what I've found online `atomic_store` and `=` are equivalent for atomics.  If we use `atomic_store_explicit` here we can relax the memory ordering requirements a bit by specifying `memory_order_relaxed`.

In other words, I'm not sure there is a more efficient way to initialize the atomics.

Reply:
@wilbaker i'm honestly not super familar in this space either.  Considering you've tested the build performance and didn't notice any change, what we have should be fine.  If @jrbriggs  has experience in this area it would be good to hear his thoughts.

Reply:
It's been a while since I've been thinking about lockless programming in C++, but the patterns here make sense to me.